packages/core/src/card/Card.tsx

```diff
@@ -56,6 +61,27 @@ export const Card = forwardRef<HTMLDivElement, CardProps>(
       window: targetWindow,
     });
 
+    const cardRef = useRef<HTMLDivElement | null>(null);
+    const handleRef = useForkRef(cardRef, ref);
+
+    useLayoutEffect(() => {
+      if (!cardRef.current) return;
+
+      const observer = new MutationObserver((mutations) => {
+        for (const mutation of mutations) {
+          for (const addedNode of mutation.addedNodes) {
+            console.log(addedNode);
```

Comment:
Is the aim here to make it so that if there's no children (subtree false? childList false?), add the `noContentComponent` class suffix?

Reply:
Yeah, this was really just to show the poc